File: Aula01/exer1.1/ex_1.1_P.php
<?php
echo "Digite seu nome: ";
$nome = trim(fgets(STDIN));

echo "Digite um número inteiro: ";
$numero = (int) trim(fgets(STDIN));

if ($numero > 0) {
    $resultado = "positivo";
} elseif ($numero < 0) {
    $resultado = "negativo";
} else {
    $resultado = "zero";
}

echo "$nome, o número $numero é $resultado.\n";
?>

File: Aula01/exer1.1/ex_1.1_P.py
# ==========================================================
# EXERCÍCIO GUIADO - PYTHON
# Leitura de nome e número inteiro
# Classificação do número como positivo, negativo ou zero
# ==========================================================

# Entrada de dados
nome = input("Digite seu nome: ")
numero = int(input("Digite um número inteiro: "))

# Processamento e saída
if numero > 0:
    print(nome + ", o número digitado é positivo.")
elif numero < 0:
    print(nome + ", o número digitado é negativo.")
else:
    print(nome + ", o número digitado é zero.")